src/Action/ExceptionAction.php

```diff
@@ -80,6 +83,12 @@ public function __invoke(FlattenException $exception, Request $request): Respons
         $headers['X-Content-Type-Options'] = 'nosniff';
         $headers['X-Frame-Options'] = 'deny';
 
-        return new Response($this->serializer->serialize($exception, $format['key']), $exception->getStatusCode(), $headers);
+        $response = new Response(null, $exception->getStatusCode(), $headers);
+
+        if (!$this->debug && $response->isServerError()) {
+            $exception->setMessage(Response::$statusTexts[$response->getStatusCode()]);
```

Comment:
@meyerbaptiste @teohhanhui @soyuka , what about the comment of @meyerbaptiste "And avoid the FlattenException::setStatusCode() in the same time" (means move it in ErrorNormalizer) ? 
The Response with status code is dealed by the ExceptionAction, not by ErrorNormalizer.... 

I feel confused, cause in the same time, We alter the exception status code in ExceptionAction and the exception message in ErrorNormalizer...